packages/core/src/memory/types.ts

```diff
@@ -1,12 +1,14 @@
-import type { AssistantContent, CoreMessage, EmbeddingModel, ToolContent, UserContent } from 'ai';
+import type { AssistantContent, CoreMessage, EmbeddingModel, LanguageModelV1, ToolContent, UserContent } from 'ai';
```

Comment:
@DanielSLew If you use DynamicArgument<MastraLanguageModel>, you don't need to import LanguageModelV1.

```suggestion
import type { AssistantContent, CoreMessage, EmbeddingModel, ToolContent, UserContent } from 'ai';
```